### PR TITLE
Issue 65 Fix

### PR DESCRIPTION
**Pull request by YoussefHenna**

### CODE REVIEW

File: src/components/AspectRatio.web.tsx [hunk 1]
```diff
@@ -1,50 +1,40 @@
 import React from "react";
 import {
   View,
-  createElement,
   StyleSheet,
-  LayoutChangeEvent,
+  LayoutRectangle,
```
Comment (peterpme) [inline src/components/AspectRatio.web.tsx:5]: what does this do?

File: src/components/AspectRatio.web.tsx [hunk 2]
```diff
@@ -1,50 +1,40 @@
...
 interface Props {
   children?: React.ReactNode;
-  onLayout?: (event: LayoutChangeEvent) => void;
-  ratio: number;
   style?: StyleProp<ViewStyle>;
 }
 
-const AspectRatio: React.FC<Props> = ({ children, onLayout, ratio, style }) => {
-  const percentage = 100 / ratio;
+const AspectRatio: React.FC<Props> = (props) => {
```
Comment (peterpme) [inline src/components/AspectRatio.web.tsx:15]: this component came from [react-native-web](https://github.com/necolas/react-native-web/issues/427#issuecomment-352474031)

do you know what differences your changes will make?

File: src/components/AspectRatio.web.tsx [hunk 3]
```diff
@@ -1,50 +1,40 @@
...
-  LayoutChangeEvent,
+  LayoutRectangle,
   StyleProp,
   ViewStyle,
 } from "react-native";
 
 interface Props {
   children?: React.ReactNode;
-  onLayout?: (event: LayoutChangeEvent) => void;
-  ratio: number;
```
Comment (peterpme) [inline src/components/AspectRatio.web.tsx:14]: i think we need ratio here still, no?

File: src/components/AspectRatio.web.tsx (hunk 1 above)
Comment (YoussefHenna) [reply src/components/AspectRatio.web.tsx:5]: Just an object with width and height

File: src/components/AspectRatio.web.tsx (hunk 2 above)
Comment (YoussefHenna) [reply src/components/AspectRatio.web.tsx:15]: It didn't seem to work at all, aspect ratio refused to work. Don't worry though I didn't make this up from my mind, I got it from [expo's website](https://baconbrix.gitbook.io/react-native-web/styling/aspect-ratio) just added some types to it to be valid typescript. This one works as intended. Maybe the old approach is outdated in newer expo versions.

File: src/components/AspectRatio.web.tsx (hunk 3 above)
Comment (YoussefHenna) [reply src/components/AspectRatio.web.tsx:14]: It uses ratio directly provided in the style, no need for additional prop

**Pull request merged**